etc/helm/pachyderm/envoy.yaml

```diff
@@ -0,0 +1,225 @@
+overload_manager:
+  refresh_interval: 0.25s
+  resource_monitors:
+    - name: "envoy.resource_monitors.fixed_heap"
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.resource_monitors.fixed_heap.v3.FixedHeapConfig
+        max_heap_size_bytes: 500000000 # 500MiB
+  actions:
+    - name: "envoy.overload_actions.shrink_heap"
+      triggers:
+        - name: "envoy.resource_monitors.fixed_heap"
+          threshold:
+            value: 0.95
+    - name: "envoy.overload_actions.stop_accepting_requests"
+      triggers:
+        - name: "envoy.resource_monitors.fixed_heap"
+          threshold:
+            value: 0.98
+admin:
+  access_log:
+    - name: envoy.access_loggers.stderr
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StderrAccessLog
+  address:
+    socket_address: { address: 0.0.0.0, port_value: 9901 }
+static_resources:
+  listeners:
+    - name: public-listener
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 80 }
+      per_connection_buffer_limit_bytes: 32768
+      traffic_direction: INBOUND
+      filter_chains:
+        - filters:
+            - name: envoy.http_connection_manager
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                access_log:
+                  - name: envoy.access_loggers.stdout
+                    typed_config:
+                      "@type": type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StdoutAccessLog
+                codec_type: auto
+                common_http_protocol_options:
+                  idle_timeout: 60s
+                  headers_with_underscores_action: REJECT_REQUEST
+                http_protocol_options:
+                  accept_http_10: false
+                http2_protocol_options:
+                  max_concurrent_streams: 100
+                  initial_stream_window_size: 65536 # 64 KiB
+                  initial_connection_window_size: 1048576 # 1 MiB
+                stream_idle_timeout: 60s
+                request_timeout: 60s
+                use_remote_address: true
+                stat_prefix: pachyderm_envoy_http
+                route_config:
+                  virtual_hosts:
+                    - name: any
+                      domains: ["*"]
+                      routes:
+                        - match:
+                            prefix: /
+                            grpc: {}
```

Comment:
If only routing grpc was always this easy 😅

Reply:
Yeah they thought of this use case in advance 😂